python/README.md

```diff
@@ -40,6 +45,49 @@ else:
     print('Timed out before predicate was true')
 ```
 
+Refer to the available methods of the `AICA` client class for more advanced usage.
+
+## Network configuration
+
+By default, the API server of AICA Core is available on the default address `localhost:8080`. Depending on the network
+configuration, the URL or port number of the AICA Core instance may be different.
+
+For example, when using AICA Launcher on macOS, the API is bound to a different, randomly generated port to avoid
```

Comment:
There is no other way than that?

Reply:
Not at the moment. However, in a production environment, you shouldn't run Core through Launcher (IMO) so I don't think it should be a big issue for final users.